stress/shell/src/main/java/alluxio/stress/cli/StressMasterBench.java

```diff
@@ -386,12 +386,22 @@ private void runInternal() throws Exception {
       }
       CommonUtils.sleepMs(waitMs);
 
-      while (!Thread.currentThread().isInterrupted()
-          && ((!useStopCount && CommonUtils.getCurrentMs() < mContext.getEndMs())
-              || (useStopCount && mContext.mCounter.get() < mParameters.mStopCount))) {
+      long localCounter = 0;
+      while (true) {
+        if (Thread.currentThread().isInterrupted()) {
+          break;
+        }
+        if (!useStopCount && CommonUtils.getCurrentMs() >= mContext.getEndMs()) {
+          break;
+        }
+        if (useStopCount
+            && (localCounter = mContext.getCounter().getAndIncrement()) >= mParameters.mStopCount) {
```

Comment:
I think this is the correct way as following. otherwise, when `useStopCount` is `false`, the 
`(localCounter = mContext.getCounter().getAndIncrement())` part will be short-circuited and `applyOperation` will see 0 passed as arg, possibly causing issues?


```suggestion
        localCounter = mContext.getCounter().getAndIncrement());
        if (useStopCount
            && (localCounter >= mParameters.mStopCount) {
```

Reply:
You're right. This is fixed in de5f9ec.